eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -173,3 +174,16 @@ jobs:
           testRunTitle: "$(OSName) $(TestTargetFramework)"
           testResultsFormat: "VSTest"
           mergeTestResults: true
+      - task: Palmmedia.reportgenerator.reportgenerator-build-release-task.reportgenerator@4
```

Comment:
Is this task installed in our devops org?

Reply:
Yes I added it from the marketplace.